client/my-sites/plans/jetpack-plans/product-lightbox/payment-plan.tsx

```diff
@@ -19,23 +19,24 @@ type PaymentPlanProps = {
 	product: SelectorProduct;
 	showPlansOneBelowTheOther?: boolean;
 	isActive?: boolean;
+	quantity?: number | null;
 };
 const PaymentPlan: React.FC< PaymentPlanProps > = ( {
 	isMultiSiteIncompatible,
 	siteId,
 	product,
 	showPlansOneBelowTheOther = false,
 	isActive = true,
+	quantity = null,
 } ) => {
 	const translate = useTranslate();
 	const { containerRef, isCompact } = useItemPriceCompact();
 
-	const { originalPrice, discountedPrice, discountedPriceDuration, isFetching } = useItemPrice(
-		siteId,
-		product,
-		product?.monthlyProductSlug || ''
-	);
+	const { originalPrice, discountedPrice, discountedPriceDuration, isFetching, priceTierList } =
+		useItemPrice( siteId, product, product?.monthlyProductSlug || '' );
 
+	const currentTier = quantity && priceTierList.find( ( tier ) => tier.maximum_units === quantity );
```

Comment:
The only change I'd suggest here is putting quantity between the min and max
```
max >= qty >= min
```

That would feel a bit more robust as opposed to the strict equality IMO

Reply:
🤔 

I am not sure if that is necessary here. The quantity comes from a tier list that is generated from the `priceTierList` which has set values we use. I don't see any way that a value not exactly the same as an item in the list gets passed to this component. 

However, it's possible some code later on gets added where that is a possibility and uses the same component (though not likely, as this component is specific to the lightboxes). I can get that added since it's not an expensive computation and for completeness 😄